src/ledger/ledger-rules/witness-validation.md

```diff
@@ -0,0 +1,81 @@
+# Witness Validation
+
+TODO: provide a description of the ledger rules relevant to this domain specific concept.
+
+## Footgun #1: Bootstrap Witnesses
+Alonzo introduced a [`bootstrap_witness`](https://github.com/IntersectMBO/cardano-ledger/blob/d71d7923a79cfcde8cac0b5db399a5427524d06a/eras/allegra/impl/cddl-files/allegra.cddl#L314-L319), which is a different structure for witnesses from a bootstrap address. One would, perhaps reasonably, assume that all witnesses from bootstrap addresses would be provided in this list. However, that is not an assumption that can be made.
```

Comment:
You say Alonzo, but link to an allegra cddl

Reply:
It's actually Shelley.